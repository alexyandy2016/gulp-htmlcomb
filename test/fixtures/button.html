<button id="button" type="button" class="button">Button</button>
