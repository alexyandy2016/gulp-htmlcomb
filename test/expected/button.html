<button class="button" id="button" type="button">Button</button>
